peers2/found_peers_handler.go

```diff
@@ -0,0 +1,35 @@
+package peers2
+
+import (
+	"context"
+
+	"github.com/ethereum/go-ethereum/p2p/discover"
+	"github.com/ethereum/go-ethereum/p2p/discv5"
+	"github.com/status-im/status-go/peers"
+)
+
+type FoundPeersHandler interface {
+	Handle(*discv5.Node) bool
+}
+
+type AcceptAllPeersHandler struct{}
```

Comment:
Maybe remove suffix `-Handler` from concrete implementation of `FoundPeersHandler`? It'll be easier to distinguish where is implementation and where is an interface itself from the name.